Scripts/script-SplunkPySearch.yml

```diff
@@ -0,0 +1,34 @@
+commonfields:
+  id: SplunkPySearch
+  version: -1
+name: SplunkPySearch
+script: |-
+  query = demisto.args()['query']
+  rows = demisto.args()['rows']
+
+  query = query + ' | head ' + rows
+  res = demisto.executeCommand('splunk-search',{'using-brand': 'splunkpy', 'query': query})
+
+  if (res and len(res)>0 and res[0]['Contents']):
+      demisto.results({"Type": 1, "Contents": res[0]['Contents'], "ContentsFormat": "json", "EntryContext": {},
```

Comment:
Is it only for display? no need outputs?

Reply:
I saw that other enhancements scripts don't update context.
They just show outputs

Reply:
If it used only for enhancements it is ok